Register storage specifier is an error for modern compiler and generally ignored anyway.

diff --git a/libavcodec/get_bits.h b/libavcodec/get_bits.h
index fe2f6378b4..477bd11746 100644
--- a/libavcodec/get_bits.h
+++ b/libavcodec/get_bits.h
@@ -291,8 +291,8 @@ static inline void skip_bits_long(GetBitContext *s, int n)
  */
 static inline int get_xbits(GetBitContext *s, int n)
 {
-    register int sign;
-    register int32_t cache;
+    int sign;
+    int32_t cache;
     OPEN_READER(re, s);
     av_assert2(n>0 && n<=25);
     UPDATE_CACHE(re, s);
@@ -305,8 +305,8 @@ static inline int get_xbits(GetBitContext *s, int n)
 
 static inline int get_xbits_le(GetBitContext *s, int n)
 {
-    register int sign;
-    register int32_t cache;
+    int sign;
+    int32_t cache;
     OPEN_READER(re, s);
     av_assert2(n>0 && n<=25);
     UPDATE_CACHE_LE(re, s);
@@ -319,7 +319,7 @@ static inline int get_xbits_le(GetBitContext *s, int n)
 
 static inline int get_sbits(GetBitContext *s, int n)
 {
-    register int tmp;
+    int tmp;
     OPEN_READER(re, s);
     av_assert2(n>0 && n<=25);
     UPDATE_CACHE(re, s);
@@ -334,7 +334,7 @@ static inline int get_sbits(GetBitContext *s, int n)
  */
 static inline unsigned int get_bits(GetBitContext *s, int n)
 {
-    register unsigned int tmp;
+    unsigned int tmp;
     OPEN_READER(re, s);
     av_assert2(n>0 && n<=25);
     UPDATE_CACHE(re, s);
@@ -355,7 +355,7 @@ static av_always_inline int get_bitsz(GetBitContext *s, int n)
 
 static inline unsigned int get_bits_le(GetBitContext *s, int n)
 {
-    register int tmp;
+    int tmp;
     OPEN_READER(re, s);
     av_assert2(n>0 && n<=25);
     UPDATE_CACHE_LE(re, s);
@@ -370,7 +370,7 @@ static inline unsigned int get_bits_le(GetBitContext *s, int n)
  */
 static inline unsigned int show_bits(GetBitContext *s, int n)
 {
-    register unsigned int tmp;
+    unsigned int tmp;
     OPEN_READER_NOSIZE(re, s);
     av_assert2(n>0 && n<=25);
     UPDATE_CACHE(re, s);
